FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/portable/NetworkInterface/WinPCap/NetworkInterface.c

```diff
@@ -44,20 +44,20 @@ access functions. */
 

```

Comment:
```WinPCap/NetworkInterface.c``` does not seem to relate to Linux Simulator. I'd recommend having another PR to avoid confusion.